doc/level0.rst

```diff
@@ -133,8 +133,9 @@ specific format, which contains a payload protected by the setup code.
   symmetrically encrypted data, and MAY include other text above or
   below the ASCII-armored data, which MUST be ignored while
   processing.
-- Decrypting the payload should produce an ASCII-armored OpenPGP
-  transferable secret key.
+- Decrypting the payload MUST produce a multipart/mixed mime structure
+  with an optional Autocrypt:-header containing the user's prefer-encrypt state;
+  one part is an ASCII-armored OpenPGP transferable secret key.
```

Comment:
If it'd be a literal ``Autocrypt`` header we would need to distinguish it from "normal" Autocrypt header -- if a header, then maybe better use a ``Autocrypt-Settings`` or ``Autocrypt-OwnState`` header to not have to distinguish it from the "normal" one?